package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "rest-client-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --turbopack",
+    "build": "next build --turbopack",
+    "start": "next start",
+    "lint": "eslint"
```

Adding a `lint:fix` script would be very helpful, for example like this:
```suggestion
    "lint:fix": "next lint --fix"
```